microsetta_admin/templates/scan.html

```diff
@@ -305,6 +311,23 @@ <h4>Scan History</h4>
                 </td>
             </tr>
             <!-- End collapsible email template fields -->
+            {% for proj_info in projects_info %}
```

Comment:
This will change when you move the retrieval of possible observations into its own API path, but I still wanted to note it. The way the code is currently written, it would display the Sample Observations `tr` for every project a barcode is associated with. If the sample is associated with two projects, neither of which has possible observations, it just has two rows of "Sample Observations:" with nothing in the right cell.